pkg/json_probe_results.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	sce "github.com/ossf/scorecard/v4/errors"
+	"github.com/ossf/scorecard/v4/finding"
+)
+
+// JSONScorecardProbeResult exports results as JSON for flat findings without checks.
+//
+//nolint:govet
+type JSONScorecardProbeResult struct {
+	Date      string            `json:"date"`
+	Repo      jsonRepoV2        `json:"repo"`
+	Scorecard jsonScorecardV2   `json:"scorecard"`
+	Findings  []finding.Finding `json:"findings"`
+	Metadata  map[string]any    `json:"metadata"`
```

Comment:
if we're not populating the `Metadata` field yet, let's drop it from the struct. Can always add it later.

Reply:
done, thanks.